3 - PB/Documentazione esterna/Verbali/24-03-22/24-03-22.typ

```diff
@@ -0,0 +1,60 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "22/03/24",
+  subTitle: "Aggiornamento sullo stato dei lavori",
+  docType: "verbale",
+  authors: (
+    "Banzato Alessio",
+  ),
+  reviewers: (
+    "Gardin Giovanni",
+  ),
+  missingMembers: (
+    "Todesco Mattia",
+  ),
+  externalParticipants : (
+    (name: "Bassani Matteo", role: "Referente aziendale"),
+    (name: "De Sio Domenico", role: "Referente aziendale"),
+  ),
+  timeStart: "16:00",
+  timeEnd: "16:30",
+  location: "Zoom",
+);
+
+= Ordine del giorno
+- Presentazione MVP;
+- Presentazione dei pattern per la ricerca;
+- Lista dei bug;
+- Pianificazione meeting futuri.
+
+== Presentazione MVP
+
+Sono state presentate ai Proponenti le funzionalità dell'MVP sviluppate dall'ultimo incontro, in particolare:
```

Comment:
Se non erro, il proponente è sempre singolare perchè individuabile come "l'Azienda". In questo caso potresti o mettere al singolare, o parlare di "referenti", just my opinion